Agda.cabal

```diff
@@ -807,6 +807,7 @@ library
       Agda.Interaction.Options.HasOptions
       Agda.Utils.CallStack.Base
       Agda.Utils.CallStack.Pretty
+      Agda.Utils.Unsafe
```

Comment:
This module should be placed under `exposed-modules` such as all the other utilities.  (Fixing this...)